docs/updates/2018-12.md

```diff
@@ -0,0 +1,65 @@
+# December 2018
+
+In the final update for this year we would like to highlight a few updates we have made.
+
+## Evaluation Trial License
+
+To better support scalability and load testing evaluations, we have added a trial license without the limitations of the
+community edition (most notably, the limit of 5 concurrent sessions).
+
+More information can be found on the [license](https://core.qlik.com/licensing/#2-evaluation-trial-license) page.
+
+## Data model experience
+
+With the basic scope in place we have started fine tuning our data modeling tool
+[catwalk](https://github.com/qlik-oss/catwalk). Some of the items we've delivered:
+
+### Support modal state in begin selection
+
+When the user begins selecting items in a filterbox, the list is not sorted until the selection mode is ended, i.e. the
+user unselects the filterbox. Once the selection mode has ended, the selections are accepted and the filterbox is
+sorted with the selected items on top.
+
+![screenshot](../images/modal-state.gif)
```

Comment:
The gif is actually not that nice rendered in my browser -> 
![screen shot 2018-12-19 at 13 15 34](https://user-images.githubusercontent.com/25030951/50219669-3d4ad880-0390-11e9-9700-cf4018029e14.png)

Reply:
Fixed